code-review.md

```diff
@@ -0,0 +1,43 @@
+# Implementing a Strong Code Review Culture
+
+## Abstract
+
+Code review is about catching bugs, right? Wrong. Modern code review is about
+socialization, learning, and teaching.
+
+What's the true value of code review? How can I get the most out of a peer's
```

Comment:
"Code review" is countable so I think it should be phrased as one of the following:

What's the true value of a code review?
What's the true value of a code reviews?
What's the true value of a reviewing code?